tests/conftest.py

```diff
@@ -135,7 +135,7 @@ def virtualenv(tmpdir, monkeypatch):
         SRC_DIR,
         pip_src,
         ignore=shutil.ignore_patterns(
-            "*.pyc", "__pycache__", "contrib", "docs",
+            "*.pyc", "__pycache__", "contrib", "docs", "tasks", "*.txt",
```

Comment:
wondering if we'd ever have tests for tasks, but if so, we could add it back I guess